### PR TITLE
Adding tests for API and login/logout

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: cafebabel/tests/test_users.py [hunk 1]
```diff
@@ -26,3 +27,17 @@ def test_admin_has_all_roles(admin):
...
+def test_authenticated_user_can_access_login_required_page(client, user):
+    login(client, user.email, 'secret')
+    response = client.get('/profile/')
+    assert response.status_code == HTTPStatus.OK
+    assert b'<h1>testy@example.com\'s profile</h1>' in response.data
+
+
+def test_logout_user_cannot_access_login_required_page(client, user):
+    login(client, user.email, 'secret')
+    logout(client)
```
Comment (vinyll) [inline cafebabel/tests/test_users.py:41]: why login then logout?

File: conftest.py [hunk 2]
```diff
@@ -16,6 +16,7 @@ def pytest_runtest_teardown():
 
 @pytest.fixture
 def app():
+    myapp.config['WTF_CSRF_ENABLED'] = False
```
Comment (vinyll) [inline conftest.py]: Could you also modify the name of the DB so that we don't lose the manual data when running tests?

**Review by vinyll — APPROVED** [review]
r+wc

 🆒 for the tests! 🌮

File: cafebabel/tests/test_users.py (hunk 1 above)
Comment (davidbgk) [reply cafebabel/tests/test_users.py:41]: To make sure that logout is effective

**Pull request merged**